component/authentication-endpoint/src/main/webapp/smsotpError.jsp

```diff
@@ -55,6 +55,12 @@
                     errorMessage = "The code entered is expired. Authentication Failed!";
                 } else if (errorMessage.equalsIgnoreCase(SMSOTPConstants.SEND_OTP_DIRECTLY_DISABLE_MSG)) {
                     errorMessage = "User not found in the directory. Cannot proceed further without SMS OTP authentication.";
+                } else if (errorMessage.equalsIgnoreCase(SMSOTPConstants.ACCOUNT_LOCKED_ERROR)) {
+                    errorMessage = "User account is locked. Please retry later.";
+                    String unlockTime = request.getParameter("unlockTime");
+                    if (unlockTime != null) {
```

Comment:
Don't we have to check for blank or empty?

Reply:
No need to check for blank because, unlockTime query param is added only if timeToUnlock > 0.

```
if (StringUtils.isNotBlank(unlockTime)) {
    long timeToUnlock = Long.parseLong(unlockTime) - System.currentTimeMillis();
    if (timeToUnlock > 0) {
          queryParams += "&unlockTime=" + Math.round((double) timeToUnlock / 1000 / 60);
     } 
}
```